security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -12,6 +12,34 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-47"]
+== OpenShift Compliance Operator 0.1.47
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.47:
+
+* link:https://access.redhat.com/errata/RHBA-2022:0014[RHBA-2022:0014 OpenShift Compliance Operator Bug Fix and Enhancement Update]
+
+[id="compliance-operator-0-1-47-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The Compliance Operator now supports the following compliance benchmarks for the Payment Card Industry (PCI) Data Security Standard:
++
+** ocp4-pci-dss
+** ocp4-pci-dss-node
+
+* Additional rules and remediations for FedRAMP moderate are added to the OCP4-moderate, OCP4-moderate-node, and rhcos4-moderate profiles.
```

Comment:
`FedRAMP moderate` : perhaps `the FedRAMP moderate impact level`

On first read, my brain made 'moderate' into the verb.

Reply:
Yes - I understand - I will double check with dev and submit a change via a second PR due to increased urgency to get this version out.